gpiozero/boards.py

```diff
@@ -1324,6 +1324,105 @@ def __init__(self, *, pwm=False, initial_value=False, pin_factory=None):
         )
 
 
+class ModMyPiStar(LEDBoard):
+    """
+    Extends :class:`LEDBoard` for the `ModMyPi Programmable Christmas Tree Star`_:
+    a Christmas tree star with 30 white LEDs.
+
+    The 30 white LEDs can be accessed through the attributes inner and outer.
+    The inner attribute controls the five inner LEDs as a single group, while
+    the outer attribute controls the 25 outer LEDs. Those 25 LEDs can be controlled
+    individually.
+
+    Alternatively, as with all descendents of :class:`LEDBoard`, you can treat
+    the instance as a sequence of LEDs (the first element is the inner LEDs).
+
+    The ModMyPi star board pins are fixed and therefore there's no need to
+    specify them when constructing this class.
+
+    The following example turns on all the inner and outer LEDs::
+
+        from gpiozero import ModMyPiStar
+        from time import sleep
+
+        star = ModMyPiStar()
+
+        for light in star:
+            light.on()
+            sleep(1)
+
+    The following example turns on the 25 outer LEDs individually in sequence,
+    at an increasingly rapid pace, before flashing the five inner LEDs, which
+    act as a single group::
+
+        from gpiozero import ModMyPiStar
+        from time import sleep
+
+        step = 0.5
+        count = 0
+        star = ModMyPiStar(pwm=True)
+        leds = star.leds
+
+        while True:
+            if(count%26!=0):
+                leds[count%26].on()
+                sleep(step)
+                leds[count%26].off()
```

Comment:
I guess this could be "simplified" to:
```suggestion
            which_led = count % 26
            if which_led != 0:
                star.leds[which_led].on()
                sleep(step)
                star.leds[which_led].off()
```

Or you could even do:
```suggestion
            which_led = count % 25
            star.outer[which_led].on()
            sleep(step)
            star.outer[which_led].off()
```
? (note that I don't have a ModMyPiStar, so this is entirely untested!)